### PR TITLE
Update sierra template to ags 4

### PR DESCRIPTION
**Pull request by mberdev**

### CODE REVIEW

File: Sierra-style/KeyboardMovement.asc [hunk 1]
```diff
@@ -127,8 +127,8 @@ function repeatedly_execute()
       keydown = true;
     }
   }
-
-  if (IsGamePaused() || mode != eKeyboardMovementModePressing || !IsInterfaceEnabled() || player.on == 0)
+
```
Comment (ericoporto) [inline Sierra-style/KeyboardMovement.asc:132]: Remove trailing space

File: Sierra-style/KeyboardMovement.asc [hunk 2]
```diff
@@ -1,5 +1,7 @@
 // Main script for module 'KeyboardMovement'
```
Comment (ericoporto) [inline Sierra-style/KeyboardMovement.asc:2]: No need to add this comment

File: Sierra-style/room1.crm
Comment (ericoporto) [inline Sierra-style/room1.crm:1]: I believe ags4 should work if this file is deleted

**Comment by ericoporto** [comment]
Also I think it's fine to have everything in a single commit, plus please add a prefix like `Sierra: ` in commits affecting sierra templates. See how our commit messages are usually written here: https://github.com/adventuregamestudio/ags-template-source/commits/master/

File: Sierra-style/KeyboardMovement.asc (hunk 2 above)
Comment (mberdev) [reply Sierra-style/KeyboardMovement.asc:2]: I disagree, as per https://www.adventuregamestudio.co.uk/forums/index.php?msg=636664304
Let's debate it in that same thread.

File: Sierra-style/room1.crm
Comment (mberdev) [reply Sierra-style/room1.crm:1]: I have deleted it but be aware that AGS 4 alpha 11 recreates the file as soon as the game gets built/run.

**Pull request merged**